core/src/avm1/function.rs

```diff
@@ -143,6 +143,136 @@ impl<'gc> Avm1Function<'gc> {
     pub fn register_count(&self) -> u8 {
         self.register_count
     }
+
+    fn debug_string_for_call(&self, name: ExecutionName<'gc>, args: &[Value<'gc>]) -> String {
+        let mut result = match self.name.map(ExecutionName::Dynamic).unwrap_or(name) {
+            ExecutionName::Static(n) => n.to_owned(),
+            ExecutionName::Dynamic(n) => n.to_utf8_lossy().into_owned(),
+        };
+        result.push('(');
+        for i in 0..args.len() {
+            result.push_str(args.get(i).unwrap().type_of());
+            if i < args.len() - 1 {
+                result.push_str(", ");
+            }
+        }
+        result.push(')');
+        result
+    }
+
+    fn is_this_inherited(&self) -> bool {
+        self.flags
+            .intersects(FunctionFlags::PRELOAD_THIS | FunctionFlags::SUPPRESS_THIS)
+    }
```

Comment:
This seems a little overkill to as a function to me, as opposed to just using it inline.